src/components/Landing/Bounties.tsx

```diff
@@ -29,6 +29,7 @@ export default function Bounties() {
 
   function handleLearnMoreBtnClick() {
     trackGoogleAnalyticsEvent('Landing', 'Learn more button click', 'Learn more Button')
+    window.open('https://docs.protocol.land', '_blank')
```

Comment:
Maybe we could send them directly into that section inside the docs?
[PL Docs - Create a new bounty](https://docs.protocol.land/working-with-issues/create-a-new-bounty)

Reply:
Yes, i will update this link.

Reply:
@7i7o I updated the link.